src/client_side_reply.cc

```diff
@@ -1562,17 +1559,10 @@ clientReplyContext::buildReplyHeader()
         Auth::UserRequest::AddReplyAuthHeader(reply, request->auth_user_request, request, http->flags.accel, 0);
 #endif
 
-    /* Append X-Cache */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE, "%s from %s",
-                      is_hit ? "HIT" : "MISS", getMyHostname());
-
-#if USE_CACHE_DIGESTS
-    /* Append X-Cache-Lookup: -- temporary hack, to be removed @?@ @?@ */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE_LOOKUP, "%s from %s:%d",
-                      lookup_type ? lookup_type : "NONE",
-                      getMyHostname(), getMyPort());
-
-#endif
+    httpHeaderPutStrf(hdr, Http::HdrType::CACHE_STATUS, "%s;%s",
+                     uniqueHostname(),
+                     (is_hit ? "hit" : "fwd=miss")
```

Comment:
> I am sticking to the values easily available in existing code and leaving all details that require logic changes to other parts of Squid out of scope.

If I find an easy way to report `fwd=stale` that does not require logic changes to other parts of Squid, would you accept it?

Reply:
sure

Reply:
Done in d9bb1a3. I have tested only a few outcomes. Please adjust as needed.

BTW, we are mishandling 412 errors, including reporting them as hits, but I do not think we should fix that bug in _this_ PR. This will probably be addressed with a proper fix for bugs [1000](https://bugs.squid-cache.org/show_bug.cgi?id=1000) and [2096](https://bugs.squid-cache.org/show_bug.cgi?id=2096).

```
HTTP/1.1 412 Precondition Failed
X-Squid-Error: ERR_PRECONDITION_FAILED 0
Age: 1598638217
Warning: 113 squid/6.0.0-VCS "This cache hit is still fresh and more than 1 day old"
Cache-Status: protofroot;hit;detail=match
```